app-virtualization/qemu/01-shared/build

```diff
@@ -125,10 +125,18 @@ pushd "${SRCDIR}/src.system/build"
 		--extra-cflags="${CFLAGS}"
 
 	abinfo "Building..."
-	make
+	make -j
 
 	abinfo "Installing to PKGDIR..."
 	make DESTDIR="${PKGDIR}" install
+
+	abinfo "Move QEMU guest agent out of package..."
+	mkdir -p "$SRCDIR"/guest-agent
+	mv -v "$PKGDIR"/usr/bin/qemu-ga "$SRCDIR"/guest-agent
```

Comment:
Consider merging some of the `mv`s, as only the last operand would be considered "target."